partner_sale_address/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (C) 2020-Today: Odoo Community Association (OCA)
```

Comment:
And do not put -Today in the copyright line. The last year, should be the year that the latest changes were made.